### PR TITLE
Update certification renewal to reflect roles vs badges

### PR DESCRIPTION
**Pull request by karenword**
The guiding purpose of this update is to align this process with changes in the way AMY, The Carpentries database, handles status with regard to certification. The addition of "roles" makes it possible for us to change status from Active to Alumni *without removing or altering a badge.* Rather than renewing certification, this process now reflects a renewal of Active status, retaining permanent certification.

Additional language changes improve clarity and accuracy with regard to process implementation. 

There are NO changes to the requirements or necessary actions for Trainers.

This requires ratification by the Trainer community and should not be merged until that process is complete.

### CODE REVIEW

**Review by jcoliver — APPROVED** [review]

**Review by paocorrales — APPROVED** [review]

**Comment by marklcrowe** [comment]
I don't have an approve button on this PR for some reason, so am just adding my approval of the proposed changes as a comment.

File: topic_folders/instructor_training/duties_agreement.md [hunk 1]
```diff
@@ -12,31 +12,39 @@ discussions about Instructor training. Trainers have opportunities to influence
 community, through leadership roles. Trainers often play leadership roles in their home communities, mentoring local Instructors, and modeling best practices.
 
 
-##### Certification Renewal Process
+##### Active Status Renewal Process
 
-Version 1.0.0 -- Approved 16 February, 2022
+Version 1.0.1 -- Draft
```
Comment (brownsarahm) [inline topic_folders/instructor_training/duties_agreement.md:17]: will we put the date in here when approved?

**Review by brownsarahm — APPROVED** [review]

File: topic_folders/instructor_training/duties_agreement.md (hunk 1 above)
Comment (karenword) [reply topic_folders/instructor_training/duties_agreement.md:17]:
```suggestion
Version 1.0.1 -- Approved 8 February, 2023
```

**Comment by karenword** [comment]
This update has been approved by the Trainer community and is ready to merge

**Pull request merged**